StartupSession/Link/Create.aplf

```diff
@@ -69,7 +69,12 @@ NOHOLD:
                  :EndIf
                  emptyns←1
              :Else
-                 emptyns←0∊⍴U.ListNames nsref←⍎ns
+                 :If 0≠≢names←U.ListNames nsref←⍎ns
```

Comment:
Too much going on here, and this ends up doing the same as `emptyns←` below. So I propose:
```apl
emptyns←0=≢names←U.ListNames nsref←⍎ns
:If ~emptynames
```
…
```apl
    emptyns←0=≢names
:EndIf
```